changelog/@unreleased/pr-1083.v2.yml

```diff
@@ -0,0 +1,5 @@
+type: fix
+fix:
+  description: Removing Feature.AD_HOC_FORMATTING from FormattingService
```

Comment:
Probably best to describe the change in terms of what the user cares about - I'd say something like "fixes bug where generated code or automated refactoring is formatted incorrectly by removing `AD_HOC_FORMATTING`"